spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -156,7 +156,7 @@ module MiqAeServiceSpec
       before { NotificationType.seed }
 
       let(:options) { {} }
-      let(:workspace) { double("MiqAeEngine::MiqAeWorkspaceRuntime", :root => options) }
+      let(:workspace) { double("MiqAeEngine::MiqAeWorkspaceRuntime", :root => options, :ae_user => user) }
```

Comment:
@tinaafitz if we change this to
```
      let(:workspace) do
        double("MiqAeEngine::MiqAeWorkspaceRuntime", :root => options,
                                                     :persist_state_hash => {}, 
                                                     :ae_user => user)
      end 
```

then you can remove all the allows for persist_state_hash and ae_user.